cpu/stm32/stmclk/stmclk_common.c

```diff
@@ -49,9 +49,6 @@
 #define RCC_CSR_LSIRDY          RCC_CSR_LSI1RDY
 #endif
 
-#ifndef CLOCK_HSE
-#define CLOCK_HSE   (0U)
-#endif
```

Comment:
Git grep still shows a bunch of occurrences of this define, are you sure it is unused?

Reply:
The define is still used but doesn't have the same semantic. CLOCK_HSE contains the frequency of the high speed external and is always defined in the clock configurations (even if there's no HSE).
The existence of the HSE is handled by `CONFIG_BOARD_HAS_HSE` now.

`CLOCK_HSE is now unused in stmclk_common.c` that's why it's removed from there.

Reply:
There are also remaining of CLOCK_HSE in clk_conf and doc.txt but those will be handled in separate PRs (same as CLOCK_LSE).

Reply:
Ok, if it was used somewhere I guess murdock would flag it.